server/src/main/scala/manifests/ManifestLookup.scala

```diff
@@ -102,12 +125,41 @@ case class ManifestLookup(tables: Tables)
         }
   }
 
+  private def historicManifestSearchForPaxCount(uniqueArrivalKey: UniqueArrivalKey,
```

Comment:
This function is identical to `historicManifestSearch` apart from one line. Would be good to refactor to a higher order function to avoid repetition